### PR TITLE
Adding MERGE command instructions when updating contact

### PR DESCRIPTION
**Pull request by luccanog**

### CODE REVIEW

File: source/includes/en/documents/contact.md [hunk 1]
```diff
@@ -26,4 +26,5 @@ Represents an contact saved in Blip.
 | shareAccountInfo | Indicates if the roster owner wants to share account information with the contact. If  true, the server provides a  get delegation permission to the contact identity into the roster owner  account resource. The default value is  true.   | boolean |
 | group            | The contact's group name.                                                                                                                                                                                                             | boolean |
 | lastMessageDate            | The contact's last interaction.                                                                                                                                                                                                             | datetimeoffset |
-| taxDocument | The client's identification document code. | string |
\ No newline at end of file
+| taxDocument | The contact's identification document code. | string |
```
Comment (mathiasfk) [inline source/includes/en/documents/contact.md:29]: "document number" sounds more natural to me

Comment (mathiasfk) [inline source/includes/en/documents/contact.md:29]: Why did you change from "client" to "contact" only here? The terminology should be consistent throughout the documentation.

File: source/includes/en/extensions/contacts.md [hunk 2]
```diff
@@ -174,28 +177,171 @@ namespace Extensions
...
-| **cellPhoneNumber** | **Optional** The client's cell phone number (string).                               | `"5531999999999"`                                            |
-| **timezone**        | **Optional** The client's timezone id (int).                                        | `-3`                                                         |
-| **culture**         | **Optional** The client's culture info (string).                                    | `"pt-br"`                                                    |
-| **extras**          | **Optional** The client's extra informations.                                       | `{"customerExternalId": "41231", "cpf": "00000000000" }`     |
-| **source**          | **Optional** The client's source (channel) info (string). Check [here](/#channels). | `"Facebook Messenger"`                                       |
+<aside  class="notice">
+The <strong>SET</strong> command is recommended to add new contacts. 
+When updating a contact using the <strong>SET</strong> command, not passing one of the contact's existing property in the request will delete it.
```
Comment (mathiasfk) [inline source/includes/en/extensions/contacts.md:186]: properties

File: source/includes/en/extensions/contacts.md [hunk 3]
```diff
@@ -174,28 +177,171 @@ namespace Extensions
...
+<br><br>
+
+Examples:
+<br>
+If the contact already has the <i>address</i> property and you don't pass It in the request, the contact's <i>adress</i> information <strong>is going to be deleted.</strong>
```
Comment (mathiasfk) [inline source/includes/en/extensions/contacts.md:191]: I don't think "It" should be uppercase here.

File: source/includes/en/extensions/contacts.md [hunk 4]
```diff
@@ -174,28 +177,171 @@ namespace Extensions
...
+<br>
+<br>
+If the contact has (or has not) the <i>name</i> property and you send It in the request, the contact's <i>name</i> value <strong>will be updated.</strong>
```
Comment (mathiasfk) [inline source/includes/en/extensions/contacts.md:194]: Same here

Comment (mathiasfk) [inline source/includes/en/extensions/contacts.md:194]: I think that "(or has not)" doesn't really help the example in this case. In fact it makes it a bit more confusing.

File: source/includes/en/extensions/contacts.md [hunk 5]
```diff
@@ -174,28 +177,171 @@ namespace Extensions
...
+                Source = "Facebook Messenger"
+            };
+
+            await _contactExtension.MergeAsync(identity, contact, cancellationToken);
+        }
+    }
+}
+```
+
+In order to update information about a chatbot's client, it is possible to save and update data using **contacts extension**.
```
Comment (mathiasfk) [inline source/includes/en/extensions/contacts.md:332]: I feel like this text is repeated. Is it in the right place?

**Review by mathiasfk — CHANGES_REQUESTED** [review]

File: source/includes/en/extensions/contacts.md (hunk 5 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md:332]: In the first text, it's written `store `intead of `update`. Do you have any suggestion to improve it?

File: source/includes/en/documents/contact.md (hunk 1 above)
Comment (luccanog) [reply source/includes/en/documents/contact.md]: solved

File: source/includes/en/extensions/contacts.md (hunk 2 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: fixed

File: source/includes/en/extensions/contacts.md (hunk 3 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: sure

File: source/includes/en/extensions/contacts.md (hunk 4 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: I tried something, please check out

File: source/includes/en/extensions/contacts.md (hunk 5 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: I tried something, please check out

File: source/includes/en/extensions/contacts.md (hunk 4 above)
Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: I tried to improve it, please check out

Comment (luccanog) [reply source/includes/en/extensions/contacts.md]: fixed

File: source/includes/en/documents/contact.md (hunk 1 above)
Comment (luccanog) [reply source/includes/en/documents/contact.md]: I've updated the other table to use the "contact" word

File: source/includes/en/documents/contact.md [hunk 6]
```diff
@@ -26,4 +26,5 @@ Represents an contact saved in Blip.
 | shareAccountInfo | Indicates if the roster owner wants to share account information with the contact. If  true, the server provides a  get delegation permission to the contact identity into the roster owner  account resource. The default value is  true.   | boolean |
 | group            | The contact's group name.                                                                                                                                                                                                             | boolean |
 | lastMessageDate            | The contact's last interaction.                                                                                                                                                                                                             | datetimeoffset |
-| taxDocument | The client's identification document code. | string |
\ No newline at end of file
+| taxDocument | The contact's identification document number. | string |
```
Comment (rodrigoxavier) [inline source/includes/en/documents/contact.md:29]: In my opinion that sentence could be better, I guess is confusing to understand if you're a new person reading this.
I suggest something like: `The contact's unique identification. For example, document number.`

Comment (luccanog) [reply source/includes/en/documents/contact.md:29]: There is another table in another file with examples. This table is only the properties simples descriptions

**Comment by yuri-aeks** [comment]
Could you update the "run postman collection" buttom? Only the share link has to be updated

File: source/includes/en/documents/contact.md (hunk 6 above)
Comment (luccanog) [reply source/includes/en/documents/contact.md:29]: @rodrigoxavier  Please, check the contacts.md file , and tell me what do you think.

**Review by JoaoCMotaJr — APPROVED** [review]

**Review by mathiasfk — APPROVED** [review]

**Pull request merged**